### ISSUE TITLE
git commit --amend

### ISSUE THREAD

**Issue by professor** [kejadlen/git-together#20]
It appears that amend is adding multiple sign-offs.

Maybe this is due to the auto-rotation of the author and committer. We like the auto-rotation of the author and committer.

**Comment by kejadlen** [comment]
This happens because git-together passes the `--signoff` command to `commit` and `merge`. (That's what adds the `Signed-off-by` text in the commit message.) A possible (and probably pretty safe) fix would be to not pass `--signoff` when `--amend` is in the argument list.

**Comment by professor** [comment]
That is cool. I'm told that git-duet does the same thing. It is annoying for both tools =)

**Comment by khuddlefish** [comment]
In addition to adding multiple "Signed-off-by" lines, it also resets the committer variable. In the above commit, this would say something like "Larry committed with Larry" when it gets pushed to github and Todd would actually not be referenced. This seems like less than ideal to me.

**Comment by khuddlefish** [comment]
@kejadlen Would it be possible to go ahead and remove the --signoff when amend is in the argument list? Having it there is definitely causing us some pain.

**Comment by kejadlen** [comment]
Yup, totally - this is high up on my todo list, but I haven't gotten around to working on git-together in a while. Next thing I do, I promise! :)

**Comment by khuddlefish** [comment]
Awesome, thank you :)

**Comment by xinzweb** [comment]
Just a suggestion, if we don't alias the `git` with `git-together`, then `git commit --amend` will work. In most scenarios, only the first commit should use the `git-together commit` to rotate the author and committers. Thanks.
